pkg/messaging/direct_messaging.go

```diff
@@ -336,6 +336,21 @@ func (d *directMessaging) invokeRemoteStream(ctx context.Context, clientV1 inter
 	}()
 	r := req.RawData()
 	reqProto := req.Proto()
+
+	// If there's a message in the proto, we remove it from the message we send to avoid sending it twice
+	// However we need to keep a reference to those bytes, and if needed add them back, to retry the request with resiliency
+	// We re-add it when the method ends to ensure we can perform retries
+	messageData := reqProto.GetMessage().GetData()
```

Comment:
Can `GetMessage()` or `GetData()` return nil?

Reply:
All the `Get*` methods have built-in nil checks:

https://github.com/dapr/dapr/blob/fa55f33e4cf7b95121a361adbbacf20ad0d9b58c/pkg/proto/common/v1/common.pb.go#L330-L342